tests/packagedcode/data/opam/sample2/sample2.opam

```diff
@@ -0,0 +1,25 @@
+pam-version: "2.0"
+maintainer:   "dev@ocsigen.org"
+authors:      "Ocsigen team"
+bug-reports:  "https://github.com/ocsigen/js_of_ocaml/issues"
+homepage:     "http://ocsigen.github.io/js_of_ocaml"
+dev-repo:     "git+https://github.com/ocsigen/js_of_ocaml.git"
+synopsis:     "Compiler from OCaml bytecode to Javascript"
+description: """
+Js_of_ocaml is a compiler from OCaml bytecode to JavaScript.
+It makes it possible to run pure OCaml programs in JavaScript
+environment like browsers and Node.js
+"""
+
+name: "js_of_ocaml"
+
+build: [["dune" "build" "-p" name "-j" jobs]]
+
+depends: [
+  "ocaml" {>= "4.02.0"}
+  "dune" {>= "1.11.1"}
+  "ocaml-migrate-parsetree" {>= "1.4"}
+  "ppx_tools_versioned" {>= "5.2.3"}
+  "uchar"
+  "js_of_ocaml-compiler" {= version}
+]
```

Comment:
Can you make sure you also handle everything in a repository opam file such as https://github.com/ocaml/opam-repository/blob/master/packages/js_of_ocaml/js_of_ocaml.3.7.0/opam ? in particular:
```
x-commit-hash: "d50221f1cf19f7637dfca7407762a85dcd420f46"
url {
  src:
    "https://github.com/ocsigen/js_of_ocaml/releases/download/3.7.0/js_of_ocaml-3.7.0.tbz"
  checksum: [
    "sha256=dcf4ffea23d4a2b2709c75bf5c7e2de355897bcfef7081d1569efe41a7638667"
    "sha512=de3fcd7b2e0a7fdd074a236efa759178888559d28db35d4431a342567b4a068bb6581545bdfca02bacdf23e08499119879d37e71c5b0e860d79252116430160e"
  ]
}
```

See also: 
https://github.com/ocaml/opam-repository/blob/master/packages/ANSITerminal/ANSITerminal.0.8/opam or https://github.com/ocaml/opam-repository/blob/b7d9c9cb5fd12c73d6cf7ca8cf05bf6a4372a8d7/packages/libtorch/libtorch.1.1.0/opam and https://github.com/ocaml/opam-repository/blob/479466a9af45ed428e382c6c13348344463c078b/packages/digestif/digestif.0.7.2/opam#L60

Reply:
Note the checksums and maintainers variations as well as the download URLs an commit hashes

Reply:
Ah, I missed these. I have added them now.